admin/cordova_sandbox.sb

```diff
@@ -0,0 +1,106 @@
+(version 1) ; Fri Jan  6 12:01:53 2017
+
+; I M P O R T A N T:
+; /private/tmp/foobar below needs to be changed with the folder that holds the source code for Cordova project
```

Comment:
I think that's part of integration work. 
When we do the integration, we're gonna decide if we just copy paste the file or generate it and replace stuff. Let's leave it like that.